### ISSUE TITLE
Projects page

### ISSUE THREAD

**Issue by francopestilli** [brain-life/core-warehouse#19]
Perhaps each project page should have a Pulse or Graphs section such as on github.com?

**Comment by soichih** [comment]
Yes, that's a very good model. I am thinking we should display something like

* Total number of datasets stored (per subject / study(?) )
* For public datasets, number of time datasets are downloaded / data-transfer size
* Recent activity (dataset archived, applications ran using the datasets)

We should also add more project specific information such as

* PI (or just rename "Admin" to "PI"?)
* Investigator (or just rename "Members" to "Investigator"?)
* Preferred citation (auto generate the default)
* (Generate DOI) button
* MARKDOWN style project documentation?

**Comment by francopestilli** [comment]
Those seem all Yes!
